docs/source/user-guide/general/ci_cd.rst

```diff
@@ -0,0 +1,130 @@
+=================================
+Using Deepchecks In CI/CD
+=================================
+
+CI/CD is a software engineering concept that is used to streamline the process of building, testing and deploying
+software products. CI/CD can also be utilized for the ML model lifecycle - to streamline the process of model training
+(and retraining), model validation and model deployment. This in turn reduces the risk of errors and improve the overall
+quality of the model.
+
+For example, CI/CD in machine learning can be used in different steps such as:
+
+* Data integrity validation: When the data used for training is collected via automatic processes and pipelines,
+  it is possible the data will contain errors and problems we haven't encountered before, either due to a bug in the
+  collection process or due to a change in the data source.
+  Examples of such problems include: missing values, outliers samples, high correlation between features,
+  label imbalance, etc.
+* Datasets comparison: In various scenarios it is needed to validate that there isn't any leakage or drift between 2
+  datasets. For example when doing a time based split of the data, there is increased chance that the split will create
+  a drift. Additional example can be for a periodic model retraining, where we might want to compare the new dataset
+  to the previous.
+* Model training: The model is trained on the validated data set.
```

Comment:
What is the CICD usecase here?

Reply:
it's a general usecase, not deepchecks-y one